### PR TITLE
Lab8

### PR DESCRIPTION
**Pull request by Maxkoz777**

### CODE REVIEW

**Comment by none-word** [comment]
Well done!

**Comment by afonyaa** [comment]
good job!

**Comment by Homa3030** [comment]
👍🏻

File: monitoring/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+version: '3.7'
+
+services:
+ app_python:
```
Comment (AKurmazov) [inline monitoring/docker-compose.yml:4]: nit: You can easily introduce memory limit by the mem_limit parameter. To choose the value use `docker stats`

**Comment by AKurmazov** [comment]
Outstanding 🔥

**Pull request closed without merging**